pkgs/data/fonts/bqn386/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "BQN386";
+  version = "unstable-2022-05-16";
+
+  src = fetchFromGitHub {
+    owner = "dzaima";
+    repo = pname;
```

Comment:
```suggestion
    repo = "BQN386";
```

Reply:
Why not reuse the `pname`?

Reply:
This is not recommended, because

1. there is few to no reason to parameterize attributes that never change
2. `pname` should be all lowercase (the only exception I can remember is SDL, for historic reasons)
3. by the point above, sometimes it is harder to convert one parameter to the other.